test/extensions/clusters/logical_dns/logical_dns_cluster_test.cc

```diff
@@ -139,6 +139,9 @@ class LogicalDnsClusterTest : public Event::TestUsingSimulatedTime, public testi
     EXPECT_EQ("", data.host_description_->locality().zone());
     EXPECT_EQ("", data.host_description_->locality().sub_zone());
     EXPECT_EQ("foo.bar.com", data.host_description_->hostname());
+    EXPECT_FALSE(data.host_description_->lastHcPassTime());
```

Comment:
i was wrong, class `RealHostDescription` is used in one [function](https://github.com/envoyproxy/envoy/blob/main/source/extensions/clusters/common/logical_host.cc#L16). So i reverted previous commit that changes visibility level and instead increased coverage in existing test. Those functions had 0% coverage. There were few more (mostly setters), but this should be good enough to pass coverage threshold.